src/WAVChunk.jl

```diff
@@ -1,15 +1,15 @@
 """
 A RIFF chunk.
 """
-type WAVChunk
+mutable struct WAVChunk
```

Comment:
This could probably be immutable.

Reply:
Is there any code that modifies the type?

Reply:
I need to check, but I believe that this type is new. I think that it came with #61 as well, which has not been part of a release.

Reply:
Came with #56, which is also not yet part of a release. So making it immutable now should be safe. If the need arises in the future, one can always make it mutable, while the opposite direction would be breaking. Should I use this opportunity then to make it immutable?

Reply:
Yes, please make the type immutable.